content/projects/connectomes.html

```diff
@@ -18,14 +18,22 @@ <h2>Connectomes</h2>
 
 <div class="doc-nav">
     <ul>
-        <li><a href="#about"><h5>About</h5></a></li>
+        <li>
```

Comment:
anything actually change in this file?

Reply:
Yes - `<a href="{{g.url('/content/projects/mri.html').path}}">Projectomes</a>.</p>` was `/projects/mri-cloud.html` before. Just turned off autoformatting as well.